demo/DemoFull.tsx

```diff
@@ -65,8 +65,35 @@ function Demo() {
                 <Grid item aria-label='example checkbox' xs={12}>
                     <ExampleOptions/>
                 </Grid>
+                { selectedExample === 'editor' ?
+                    <Grid item aria-label='editor textfield' xs={12}>
+                        <Typography variant='body1' color={editorValid === 'invalid' ? 'error' : ''}>Gosling spec is {editorValid}</Typography>
+                        <TextField 
+                            id="editor"
+                            multiline 
+                            fullWidth
+                            onChange={(event) => {
+                                setSelectedExample('editor')
```

Comment:
I think this is not needed since the value of the `selectedExample` should already be `'editor'`?

```suggestion
```

Reply:
You're right!